index.bs

```diff
@@ -14898,12 +14898,12 @@ text is “<span class="input">a1</span>”, it is tokenized as a single <emu-t
 and not as a separate <emu-t class="regex"><a href="#prod-identifier">identifier</a></emu-t> and <emu-t class="regex"><a href="#prod-integer">integer</a></emu-t>.
 If the longest possible match could match one of the above named terminal symbols or
 one of the other terminal symbols from the grammar, it must be tokenized as the latter.
-Thus, the input text “<span class="input">long</span>” is tokenized as the quoted terminal symbol
+Thus, the input text “<span class="input">long</span>” is tokenized as the monospaced terminal symbol
 <emu-t>long</emu-t> rather than an <emu-t class="regex"><a href="#prod-identifier">identifier</a></emu-t> called "<code>long</code>",
-and “<span class="input">.</span>” is tokenized as the quoted terminal symbol
+and “<span class="input">.</span>” is tokenized as the monospaced terminal symbol
 <emu-t>.</emu-t> rather than an <emu-t class="regex"><a href="#prod-other">other</a></emu-t>.
 
-The IDL syntax is case sensitive, both for the quoted terminal symbols
+The IDL syntax is case sensitive, both for the monospaced terminal symbols
 used in the grammar and the values used for
```

Comment:
Ah I misunderstood. I thought you meant the paragraphs even earlier that also talk about fonts.

Perhaps here we can simply reuse the line from above "other terminal symbols from the grammar"?